main.go

```diff
@@ -1,27 +1,20 @@
 package main
 
 import (
-	// "fmt"
-	// "os"
-
-	// tea "github.com/charmbracelet/bubbletea"
 	"fmt"
 	"os"
 
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/maximiliansoerenpollak/todotui/db"
 	"github.com/maximiliansoerenpollak/todotui/models"
-	"github.com/maximiliansoerenpollak/todotui/types"
 )
 
 func main() {
 	db.Write(db.MemData)
-	db.MemData.TaskGroups = append(db.MemData.TaskGroups, types.TaskGroup{GroupTitle: "TaskGroup 1", Tasks: []types.Task{}, GroupDescription: "Description 1"})
```

Comment:
A bit confused why this db is no longer needed?

Reply:
This line was just adding a hardcoded value to the list of taskgroups, but now we can add it through interface

Reply:
Ah perfect.